doc/source/schemas/genotypephenotype.rst

```diff
@@ -0,0 +1,172 @@
+schema.
+`SearchFeaturesResponse <https://github.com/ga4gh/schemas/blob/be171b00a5f164836dfd40ea5ae75ea56924d316/src/main/resources/avro/genotypephenotypemethods.avdl#L130>`__
+is the response from ``POST /genotypephenotype/search``, also expressed
+as JSON.
+
+The ``SearchFeaturesRequest`` and ``SearchFeaturesResponse`` records
+each have their own data structures, but they use many of the same types
+(see the 3rd table for shared data-types). Many types rely heavily on
+the concept of an
+`OntologyTerm <https://github.com/ga4gh/schemas/blob/be171b00a5f164836dfd40ea5ae75ea56924d316/src/main/resources/avro/ontologies.avdl#L10>`__
+(see end of document for discussion on usage of OntologyTerms).
+
+Use cases
+---------
+
+1) As a clinician or a genomics researcher, I may have a patient with
+   Gastrointestinal stromal tumor, GIST, and a proposed drug for
+   treatment, imatinib. In order to identify whether the patient would
+   respond well to treatment with the drug, I need a list of features
+   (e.g. genes) which are associated with the sensitivity of GIST to
+   imatinib. Suppose I am specifically interested in a gene, *KIT*,
+   which is implicated in the pathogenesis of several cancer types. I
+   could submit a query to ``/genotypephenotype/search`` with GIST as
+   the phenotype, *KIT* as the feature, and
+   `clinical study evidence <http://purl.obolibrary.org/obo/ECO_0000180>`__
+   as the evidence.
+
+In response, I will receive back a list of associations involving GIST
+and *KIT*, which I can filter for instances where imatinib is mentioned.
+URI's in the ``associations`` field could - hypothetically - be followed
+to discover that `GIST patients with wild-type *KIT* have decreased
+sensitivity to therapy with
+imatinib <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2651076/>`__.
+
+If I left both the ``genotype`` and ``evidence`` fields as ``null``, I
+would receive back all associations which involve GIST as a phenotype.
+
+2) As a non-Hodgkin's lymphoma researcher, I may know that the gene
+   *CD20* has abnormal expression in
+   `Hodgkin's lymphoma <http://purl.obolibrary.org/obo/DOID_8567>`__.
+   I might be interested in knowing whether *CD20* also has abnormal
+   expression in
+   `non-Hodgkin lymphoma <http://purl.obolibrary.org/obo/DOID_0060060>`__.
+   Therefore I could perform a query with *CD20* as a feature,
+   non-Hodgkin's lymphoma as a phenotype, and
+   `RNA sequencing <http://purl.obolibrary.org/obo/OBI_0001177>`__
+   as the evidence type.
+
+3) As a genetic counselor, I may be wondering if a mutation in one of my
+   clients' genes has ever been associated with a disease. I could then
+   do a query based on the gene name as the feature and
+   `disease <http://purl.obolibrary.org/obo/DOID_4>`__ as the
+   phenotype.
+
+For specifics of the json representations, please see the
+`server <https://github.com/ga4gh/server>`__ and
+`compliance <https://github.com/ga4gh/compliance>`__ repositories.
+
```

Comment:
Very useful doc!  It would be good to tie it into the other parts of the API. So how do you go from variants or variant annotations to G2p.